packages/rspack-dev-server/tests/__snapshots__/normalizeOptions.test.ts.snap

```diff
@@ -3,10 +3,12 @@
 exports[`normalize options snapshot additional entires should added 1`] = `
 {
   "main": [
+    "<prefix>/./placeholder.js",
+  ],
+  "undefined": [
```

Comment:
Is this expected?

Reply:
Yes, it is how webpack 5 prepend entries